deploy-board/deploy_board/webapp/cluster_view.py

```diff
@@ -458,8 +458,12 @@ def get_base_images_by_name(request):
 
 def get_base_image_info_by_name(request, name, cell):
     if name.startswith('cmp_base'):
-        base_images = baseimages_helper.get_acceptance_by_name(request, name, cell)
         with_acceptance_rs = []
+        base_images = baseimages_helper.get_acceptance_by_name(request, name, cell)
```

Comment:
by `get_acceptance_by_name`, we still rely on base image validation results from rodimus to get the list of base images?

Reply:
yes